features/column-pseudo.yml

```
@@ -0,0 +1,4 @@
+name: ::column pseudo-element
```

```suggestion
name: ::column
```

For consistency with other pseudo-elements in the repo.